packages/asset-registry/src/wrappedContracts/AssetConsumingRegistryLogic.ts

```diff
@@ -5,24 +5,24 @@ import moment from 'moment';
 
 export class AssetConsumingRegistryLogic extends GeneralFunctions {
     web3: Web3;
-    buildFile = AssetConsumingRegistryLogicJSON;
 
     constructor(web3: Web3, address?: string) {
+        const buildFile: any = AssetConsumingRegistryLogicJSON;
```

Comment:
Same comment, we're dropping the types resolved from JSON by TS here by using :any